cmake/installdata/windows/dlls.txt

```diff
@@ -1,3 +1,4 @@
 system/*.dll .
+project/BuildDependencies/${ARCH}/bin/libbluray*.jar .
```

Comment:
I can also repack libbluray to follow the new packing still, if you prefer to have it more it unified